### PR TITLE
[FLINK-16103] Translate "Configuration" page of "Table API & SQL" int…

### PR DESCRIPTION
**Pull request by Gmoonnight**
…o Chinese

## What is the purpose of the change

*(For example: This pull request makes task deployment go through the blob server, rather than through RPC. That way we avoid re-transferring them on each deployment (during recovery).)*

## Brief change log

*(for example:)*
  - *The TaskInfo is stored in the blob store on job creation time as a persistent artifact*
  - *Deployments RPC transmits only the blob storage reference*
  - *TaskManagers retrieve the TaskInfo from the blob cache*

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by wuchong** [comment]
@leonardBang , could you help to review this translation?

**Comment by leonardBang** [comment]
> @leonardBang , could you help to review this translation?
sure

File: docs/dev/table/config.zh.md [hunk 1]
```diff
@@ -22,29 +22,22 @@ specific language governing permissions and limitations
...
-In every table environment, the `TableConfig` offers options for configuring the current session.
+在每个Table环境中，`TableConfig` 提供了用于配置当前会话的选项。
 
-For common or important configuration options, the `TableConfig` provides getters and setters methods
-with detailed inline documentation.
+对于常见或者重要的配置选项，`TableConfig` 提供了具有详细内联文档的`getters`和`setters`方法。
 
-For more advanced configuration, users can directly access the underlying key-value map. The following
-sections list all available options that can be used to adjust Flink Table & SQL API programs.
+对于更加高级的配置，用户可以直接访问基础键值对。以下各节列举了所有可用于调整 Flink Table & SQL API 程序的选项。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:38]: 用户可以直接访问底层的键值 map 对象。以下章节列举了所有可用于调整 Flink Table & SQL API 程序的选项。

File: docs/dev/table/config.zh.md [hunk 2]
```diff
@@ -22,29 +22,22 @@ specific language governing permissions and limitations
...
 
-<span class="label label-danger">Attention</span> Because options are read at different point in time
-when performing operations, it is recommended to set configuration options early after instantiating a
-table environment.
+<span class="label label-danger">注意</span> 因为在执行操作时，选项会在不同的时间点被读取，所以推荐在实例化Table环境后尽早地设置配置选项。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:41]: 因为配置项会在执行操作的不同的时间点被读取，所以推荐在实例化 TableEnvironment 后尽早地设置配置项。

File: docs/dev/table/config.zh.md [hunk 3]
```diff
@@ -90,17 +83,16 @@ configuration.set_string("table.exec.mini-batch.size", "5000");
...
-The following options can be used to tune the performance of the query execution.
+以下选项可用于调整查询执行的性能。
 
 {% include generated/execution_config_configuration.html %}
 
-### Optimizer Options
+### 优化器选项
 
-The following options can be used to adjust the behavior of the query optimizer to get a better execution plan.
+以下查询可用于调整查询优化器的行为以获得更好的执行计划。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:96]: 以下选项可以用于

File: docs/dev/table/config.zh.md [hunk 4]
```diff
@@ -90,17 +83,16 @@ configuration.set_string("table.exec.mini-batch.size", "5000");
 </div>
 </div>
 
-<span class="label label-danger">Attention</span> Currently, key-value options are only supported
-for the Blink planner.
+<span class="label label-danger">注意</span> 目前，键值选项仅被Blink计划器支持。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:86]: 被Blink计划器支持-> 被 Blink planner 支持

**Comment by Gmoonnight** [comment]
@wuchong @leonardBang Thank you for taking the time to check the code for me.

File: docs/dev/table/config.zh.md [hunk 5]
```diff
@@ -90,17 +83,16 @@ configuration.set_string("table.exec.mini-batch.size", "5000");
...
 
-<span class="label label-danger">Attention</span> Currently, key-value options are only supported
-for the Blink planner.
+<span class="label label-danger">注意</span> 目前，键值选项仅被 Blink planner 支持。
 
-### Execution Options
+### 执行选项
 
-The following options can be used to tune the performance of the query execution.
+以下选项可用于调整查询执行的性能。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:90]: 调整 -> 优化

File: docs/dev/table/config.zh.md [hunk 6]
```diff
@@ -90,17 +83,16 @@ configuration.set_string("table.exec.mini-batch.size", "5000");
...
 </div>
 
-<span class="label label-danger">Attention</span> Currently, key-value options are only supported
-for the Blink planner.
+<span class="label label-danger">注意</span> 目前，键值选项仅被 Blink planner 支持。
 
-### Execution Options
+### 执行选项
 
-The following options can be used to tune the performance of the query execution.
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:98]: 调整 -> 优化
（这里指优化query）

File: docs/dev/table/config.zh.md [hunk 7]
```diff
@@ -22,29 +22,22 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-By default, the Table & SQL API is preconfigured for producing accurate results with acceptable
-performance.
+默认情况下，Table & SQL API 已经预先配置为在产生准确结果的同时性能也可接受。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:25]: > 默认情况下，Table & SQL API 已经预先配置为在产生准确结果的同时性能也可接受。

Table 和 SQL API 的默认配置能够确保结果准确，同时也提供可接受的性能。
（想了下, 我们不要直译吧，这里意译会通顺些）

File: docs/dev/table/config.zh.md [hunk 8]
```diff
@@ -22,29 +22,22 @@ specific language governing permissions and limitations
...
 
-Depending on the requirements of a table program, it might be necessary to adjust
-certain parameters for optimization. For example, unbounded streaming programs may need to ensure
-that the required state size is capped (see [streaming concepts](./streaming/query_configuration.html)).
+根据 Table 程序的要求，可能需要调整特定的参数用于优化。例如，无界流程序可能需要保证所需的状态大小是有上限的(请参阅 [流式概念](./streaming/query_configuration.html)).
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:27]: > 根据 Table 程序的要求，可能需要调整特定的参数用于优化。例如，无界流程序可能需要保证所需的状态大小是有上限的
根据 Table 程序的需求，可能需要调整特定的参数用于优化。例如，无界流程序可能需要保证所需的状态是有限的

File: docs/dev/table/config.zh.md [hunk 9]
```diff
@@ -22,29 +22,22 @@ specific language governing permissions and limitations
...
-In every table environment, the `TableConfig` offers options for configuring the current session.
+在每个 TableEnvironment 中，`TableConfig` 提供用于配置当前会话的选项。
 
-For common or important configuration options, the `TableConfig` provides getters and setters methods
-with detailed inline documentation.
+对于常见或者重要的配置选项，`TableConfig` 提供带有详细注释的 `getters` 和 `setters` 方法。
 
-For more advanced configuration, users can directly access the underlying key-value map. The following
-sections list all available options that can be used to adjust Flink Table & SQL API programs.
+对于更加高级的配置，用户可以直接访问底层的键值 map 对象。以下章节列举了所有可用于调整 Flink Table & SQL API 程序的选项。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:38]: >对于更加高级的配置，用户可以直接访问底层的键值 map 对象。以下章节列举了所有可用于调整 Flink Table & SQL API 程序的选项。

对于更加高级的配置，用户可以直接访问底层的 key-value 配置项。以下章节列举了所有可用于调整 Flink Table 和 SQL API 程序的配置项。

File: docs/dev/table/config.zh.md [hunk 10]
```diff
@@ -22,29 +22,22 @@ specific language governing permissions and limitations
...
+根据 Table 程序的要求，可能需要调整特定的参数用于优化。例如，无界流程序可能需要保证所需的状态大小是有上限的(请参阅 [流式概念](./streaming/query_configuration.html)).
 
 * This will be replaced by the TOC
 {:toc}
 
-### Overview
+### 概览
 
-In every table environment, the `TableConfig` offers options for configuring the current session.
+在每个 TableEnvironment 中，`TableConfig` 提供用于配置当前会话的选项。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:34]: `TableConfig` 提供用于当前会话的配置项。

File: docs/dev/table/config.zh.md [hunk 11]
```diff
@@ -22,29 +22,22 @@ specific language governing permissions and limitations
...
 
-For common or important configuration options, the `TableConfig` provides getters and setters methods
-with detailed inline documentation.
+对于常见或者重要的配置选项，`TableConfig` 提供带有详细注释的 `getters` 和 `setters` 方法。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:36]: 对于常见或者重要的配置项

File: docs/dev/table/config.zh.md [hunk 12]
```diff
@@ -90,17 +83,16 @@ configuration.set_string("table.exec.mini-batch.size", "5000");
 </div>
 </div>
 
-<span class="label label-danger">Attention</span> Currently, key-value options are only supported
-for the Blink planner.
+<span class="label label-danger">注意</span> 目前，键值选项仅被 Blink planner 支持。
 
-### Execution Options
+### 执行选项
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:88]: 执行配置

File: docs/dev/table/config.zh.md [hunk 13]
```diff
@@ -90,17 +83,16 @@ configuration.set_string("table.exec.mini-batch.size", "5000");
 </div>
 </div>
 
-<span class="label label-danger">Attention</span> Currently, key-value options are only supported
-for the Blink planner.
+<span class="label label-danger">注意</span> 目前，键值选项仅被 Blink planner 支持。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:86]: 键值选项 -> key-value 配置项

File: docs/dev/table/config.zh.md [hunk 14]
```diff
@@ -90,17 +83,16 @@ configuration.set_string("table.exec.mini-batch.size", "5000");
...
-### Execution Options
+### 执行选项
 
-The following options can be used to tune the performance of the query execution.
+以下选项可用于优化查询执行的性能。
 
 {% include generated/execution_config_configuration.html %}
 
-### Optimizer Options
+### 优化器选项
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:94]: 优化器配置

File: docs/dev/table/config.zh.md [hunk 15]
```diff
@@ -90,17 +83,16 @@ configuration.set_string("table.exec.mini-batch.size", "5000");
...
 
-The following options can be used to adjust the behavior of the query optimizer to get a better execution plan.
+以下选项可以用于调整查询优化器的行为以获得更好的执行计划。
```
Comment (leonardBang) [inline docs/dev/table/config.zh.md:96]: 以下配置

**Comment by Gmoonnight** [comment]
你是对的，保证翻译结果准确性的情况下，音译过来更容易理解；

官网的中文翻译往往晦涩难懂，看完模棱两可、无所获；

**Pull request merged**